camunda-commons-ui/lib/services/HttpClient.js

```diff
@@ -18,7 +18,7 @@
 'use strict';
 
 var angular = require('../../../camunda-bpm-sdk-js/vendor/angular');
-var CamSDK = require('camunda-bpm-sdk-js/lib/angularjs/index');
+var CamSDK = require('../../../camunda-bpm-sdk-js/lib/angularjs/index');
```

Comment:
This breaks the build by not using the global SDK, see https://github.com/camunda/camunda-bpm-platform/pull/1503

```suggestion
var CamSDK = require('camunda-bpm-sdk-js/lib/angularjs/index');
```